templates/repo/commit_status.tmpl

```diff
@@ -1,13 +1,13 @@
 <!-- make sure this matches the color logic in web_src/js/components/DashboardRepoList.vue -->
-{{if eq .State "pending"}}
-	{{svg "octicon-dot-fill" 18 "commit-status icon text yellow"}}
-{{end}}
-{{if eq .State "success"}}
-	{{svg "octicon-check" 18 "commit-status icon text green"}}
-{{end}}
-{{if eq .State "error"}}
+{{if eq .State 1}}
 	{{svg "gitea-exclamation" 18 "commit-status icon text red"}}
 {{end}}
-{{if eq .State "failure"}}
+{{if eq .State 2}}
```

Comment:
I believe that using direct numbers for judgment here leads to poor code readability and maintainability. My opinion is to continue using the previous code on the frontend page.When passing the state to the frontend, perform a conversion from numbers to strings.

Reply:
Ah, forgot that we have CommitStatusState.IsXXXX.
I think this is the best solution.